### PR TITLE
(MODULES-10510) Blogpost for Gemfile Update

### PR DESCRIPTION
**Pull request by sheenaajay**
This commit adds the first blog post about using the pdksync tool to perform single or multi-gem testing.

NB: this post will not display until its date has passed AND the published value in the front matter is set to true.

### CODE REVIEW

**Comment by sheenaajay** [comment]
@carabasdaniel sure thank you.

**Comment by sheenaajay** [comment]
@carabasdaniel .Add Introduction. Could you please check.

File: _posts/_posts/2020-02-06-blog-pdksync_GemTesting.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  - testing
+  - walkthrough
+tags:
+  - pdksync
+  - modules
+  - gem testing
+---
+> Introduction:
+>
+pdksync is a tool that enables you to run a pdk update against a set of defined modules. The tool checks out a module, runs the most recent version of pdk update then proceeds to create the PR for you with the changes. This is the automated tool for ensuring modules can quickly adopt changes introduced by updates to the PDK and PDK template.
```
Comment (carabasdaniel) [inline _posts/_posts/2020-02-06-blog-pdksync_GemTesting.md]: pdksync enables you to do a lot more than just pdk update against a set of defined modules

Comment (sheenaajay) [reply _posts/_posts/2020-02-06-blog-pdksync_GemTesting.md]: Sure.Thank you.

**Pull request closed without merging**